edb/server/server.py

```diff
@@ -811,36 +819,33 @@ async def run_startup_script_and_exit(self):
     async def _start_servers(self, host, port):
         nethost = await _fix_localhost(host)
 
+        sslctx = ssl.SSLContext()
+        sslctx.load_cert_chain(
```

Comment:
We should handle cert issues here and wrap them with a nice error.  Specifically, if `tls_key_file` is missing and not present in `cert_file`.